src/main/java/no/nav/veilarbdirigent/service/DialogService.java

```diff
@@ -35,19 +38,23 @@
     public Try<String> opprettDialogHvisBrukerErPermittert(AktorId aktorId) {
         Fnr brukerFnr = aktorOppslagClient.hentFnr(aktorId);
 
-        Try<BrukerRegistreringWrapper> registeringsData = veilarbregistreringClient.hentRegistrering(brukerFnr);
+        Try<Optional<BrukerRegistreringWrapper>> maybeRegistreringData = veilarbregistreringClient.hentRegistrering(brukerFnr);
+
+        if (maybeRegistreringData.isFailure()){
+            return Try.failure(maybeRegistreringData.getCause());
+        }
 
-        if (registeringsData.isFailure()){
-            return Try.failure(registeringsData.getCause());
+        if (maybeRegistreringData.get().isEmpty()) {
+            log.info("Bruker med aktorId={} er ikke registrert i arbeidssokerregistrering og vil ikke få dialog om permittering", aktorId);
+            return Try.success("Bruker ikke registrert gjennom arbeidssokerregistrering");
         }
 
-        Option<DinSituasjonSvar> svar = registeringsData
-                .toOption()
-                .flatMap(Option::of)
+        Optional<DinSituasjonSvar> svar = maybeRegistreringData.get()
                 .map(RegistreringUtils::hentBesvarelse)
                 .map(Besvarelse::getDinSituasjon);
 
-        if (svar.map(DinSituasjonSvar.ER_PERMITTERT::equals).getOrElse(false)) {
+        if (svar.map(DinSituasjonSvar.ER_PERMITTERT::equals).orElse(false)) {
+            log.info("Oppretter dialog om permittering for bruker aktorId={}", aktorId);
```

Comment:
har vi lov til å logge aktørid

Reply:
Vi gjør det omtrent alle steder. Vet at det er noen som ikke ønsker det, men pr i dag så må vi ha noe som gjør at vi kan debugge feil

Reply:
I en fjern fremtid så har vi kanskje en egen intern ID som vi kan bruke for logging

Reply:
ja, eller kanskje secure logging i kubernetes som jeg ikke husker navnet på nå